src/components/buttons/tertiary.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+export interface TertiaryButtonProps {
+  label: string | React.ReactNode;
+  onClick: () => void;
+  disabled?: boolean;
+  destructive?: boolean;
+}
+
+const TertiaryButton: React.FC<TertiaryButtonProps> = ({
+  label,
+  onClick,
+  disabled,
+  destructive,
```

Comment:
Right now, the destructive style is for the primary button defined in a different file, while for tertiary you can set it as a prop. I think we should choose only one way and be consistent on it. I think would prefer to have also a tertiary destructive in its own file. What do you think?

Reply:
I agree, that would make it more consistent. Then we will have a "primary destructive" and "tertiary destructive" button for clear naming? @raphael-arce

Reply:
@aeschi sounds great 👍